gradle.properties

```diff
@@ -1,5 +1,7 @@
 version=28.2.2
 group=com.linkedin.pegasus
+sonatypeUsername=please_set_in_home_dir_if_uploading_to_maven_central
```

Comment:
change necessary?

Reply:
from somebody else's commit, just a merge issue, you can ignore.